scout/server/blueprints/cases/static/case_images.js

```diff
@@ -162,6 +164,10 @@ function clear_svg(svg_element){
  *
  */
 function set_svg_size(svg_element, number_of_columns){
+		if(number_of_columns == 1){
```

Comment:
This looks like the perfect situation to introduce a [switch](https://www.w3schools.com/js/js_switch.asp)!

Reply:
Hihi, thanks - it is like a Pascal dialect!